cdap-docs/cdap-apps/source/tracker/index.rst

```diff
@@ -0,0 +1,441 @@
+
+  http://host:dashboard-bind-port/ns/default/tracker/home
+  
+
+Tracker is built from a system artifact included with CDAP, |literal-cask-tracker-version-jar|.
+
+
+Installation
+============
+Cask Tracker is deployed from its system artifact included with CDAP. A CDAP administrator
+does not need to build anything to add Cask Tracker to CDAP; they merely need to add two
+properties to the ``cdap-site.xml`` and the enable the the application after starting CDAP.
+
+These two properties are required in the ``cdap-site.xml`` for the audit log functionality of Tracker::
+  
+  <!-- Audit Configuration -->
+
+  <property>
+    <name>audit.enabled</name>
+    <value>true</value>
+    <description>
+      Determines whether to publish audit messages to Apache Kafka
+    </description>
+  </property>
+
+  <property>
+    <name>audit.kafka.topic</name>
+    <value>audit</value>
+    <description>
+      Apache Kafka topic name to which audit messages are published
+    </description>
+  </property>
+
+Enabling Tracker
+----------------
+To enable Tracker, go to the **Tracker UI** at 
+(in the SDK) http://localhost:9999/ns/default/tracker/home or 
+(Distributed CDAP) http://host:dashboard-bind-port/ns/default/tracker/home 
+and press the ``"Enable Tracker"`` button to deploy and start Tracker.
+
+Once pressed, the application will be deployed, the datasets created (if necessary), the flow and service
+started, and search and audit logging will become available.
+
+If you are enabling Tracker from outside the UI, you will, in addition to enabling auditing 
+in the ``cdap-site.xml`` as described above, need to:
+
+- load the artifact (|literal-cask-tracker-version-jar|)
+  
+- create the application based on the artifact, supplying a configuration file::
+    
+    {
+      "config": {
+        "auditLogKafkaConfig": {
+          "zookeeperString": "<host>:<port>/cdap/kafka"
+        }
+      }
+    }
+
+  substituting for ``<host>`` and ``<port>`` with appropriate values.
+  
+Restarting CDAP
+---------------
+As Tracker is an application running inside CDAP, it does not start up automatically when
+CDAP is restarted. Each time that you start CDAP, you will need to re-enable Tracker.
+Re-enabling Tracker does not recreate the datasets; instead, the same datasets as were
+used in previous runs are used.
+
+If you are using the audit log feature of Tracker, it is important that Tracker be enabled
+**before** you begin any other applications, or their activities will not be recorded by
+Tracker in its audit log.
+
+If the installation of CDAP is an upgrade from a previous version, all activity and
+datasets prior to the enabling of Tracker will not be available or seen in the Tracker UI.
+
+Disabling and Removing Tracker
+------------------------------
+If for some reason you need to disable or remove Tracker, you would need to:
+
+- stop the Tracker flow
+- stop the Tracker service
+- delete the Tracker application
+- delete the Tracker datasets
```

Comment:
we should clarify that restarting CDAP isn't necessary. Seems clunky to require a restart to stop/disable Tracker.